accounts/serializers.py

```diff
@@ -10,19 +11,25 @@
 
 # Allow all fields except password
 #! Danger never allow a post or put to this serializer (anyone can modify staff or admin)
-class GetUserSerializer(serializers.ModelSerializer):
+class GetUserSerializer(TaggitSerializer, serializers.ModelSerializer):
+    team = TagListSerializerField()
+
     class Meta:
         model = User
         exclude = ['password', 'is_verified']
 
 # Only allow id, email, name
-class UserPublicSerializer(serializers.ModelSerializer):
+class UserPublicSerializer(TaggitSerializer, serializers.ModelSerializer):
+    team = TagListSerializerField()
+
     class Meta:
         model = User
-        fields = ['first_name', 'last_name', 'email', 'linkedin', 'bio', 'photo', 'role', 'country']
+        fields = ['first_name', 'last_name', 'email', 'linkedin', 'bio', 'photo', 'role', 'country', 'leadership', 'team']
```

Comment:
not sure if they should be in this list

Reply:
this is a public serializer so only fields which are okay being seen in public should be added here, I think this is good